package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "scripts": {
     "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest"
```

Comment:
`NODE_OPTIONS=--experimental-vm-modules` is required to use `import` in Jest tests